service/handlers.go

```diff
@@ -20,6 +21,8 @@ import (
 	"github.com/brimdata/zed/zqe"
 )
 
+const queryStatsInterval = time.Second
```

Comment:
Nit: Put this inside handleQuery since it's unlikely it'll ever be used anywhere else.

Reply:
Where would you put this at the top of function?

Reply:
I'd just put it right before `t := time.NewTicker(queryStatsInterval)`, but top of the function works tool.